constraints.txt

```diff
@@ -149,9 +147,8 @@ mdurl==0.1.2
 mergedeep==1.3.4
 mistune==3.0.2
 mkdocs==1.5.3
-mkdocs-material==9.5.15
+mkdocs-material==9.5.16
 mkdocs-material-extensions==1.3.1
-mkdocs-mermaid2-plugin==0.6.0
```

Comment:
I vague remember @hrshdhgd that this was important to us - why am I thinking this?

Reply:
You mean the library itself or the version change?

Reply:
Didnt Sujay make mkdocs-mermaid2 a dependency on the linkml cookiecutter?

Reply:
yes , this library makes flowcharts etc in the docs.